editor/debugger/editor_profiler.cpp

```diff
@@ -351,6 +352,10 @@ void EditorProfiler::_update_frame() {
 		for (int j = m.categories[i].items.size() - 1; j >= 0; j--) {
 			const Metric::Category::Item &it = m.categories[i].items[j];
 
+			bool internal_func = (it.internal == it.total);
+			if (internal_func && m.categories[i].name == "Script Functions" && !display_internal_profiles->is_pressed()) {
```

Comment:
```suggestion
			if (internal_func && !display_internal_profiles->is_pressed() && m.categories[i].name == "Script Functions") {
```
I'd move the more expensive check later